README.md

```diff
@@ -1 +1,14 @@
-# bham1-front-end
\ No newline at end of file
+# bham1-front-end
+
+How to start the Express service application
```

Comment:
Need some info on setting up the .env file. What variables as that's a prerequisite?